pkg/deploy/graph/analysis/dc.go

```diff
@@ -50,21 +55,40 @@ dc:
 					continue dc
 				}
 
-				// 2. Build config points to image stream tag of interest
 				if bcNode, points := buildPointsToTag(g, uncastIstNode); points {
-					markers = append(markers, osgraph.Marker{
+					// A build config points to the non existent tag but the latest build has failed.
+					// Since the tag isn't yet populated, there hasn't been any successful build so far.
+					if buildNode, failed := latestBuildFailed(g, bcNode); failed {
+						markers = append(markers, osgraph.Marker{
```

Comment:
Right, I updated both this error and the warning re a bc with no builds. ptal so I can squash